src/Paket.Core/PackageResolver.fs

```diff
@@ -651,7 +651,7 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
 
     let rec step (stage:Stage) (stackpack:StackPack) compatibleVersions (flags:StepFlags) =
 
-        let fuseConflicts currentConflict priorConflictSteps =
+        let inline fuseConflicts currentConflict priorConflictSteps =
```

Comment:
@cloudRoutine Is this OK? This prevents a stack-overflow I had when debugging (compiling in DEBUG mode)